### PR TITLE
High load clusters IO requirements

### PR DESCRIPTION
**Pull request by burdandrei**

### CODE REVIEW

**Comment by burdandrei** [comment]
I adjusted the docs after my cluster running on AWS stuck twice because of IO saturation of EBS

File: website/source/guides/cluster/requirements.html.md [hunk 1]
```diff
@@ -12,14 +12,14 @@ description: |-
...
+having between 4-8+ cores, 16-32 GB+ of memory, 40-80 GB+ of **fast** disk and
 significant network bandwidth. The core count and network recommendations are to
 ensure high throughput as Nomad heavily relies on network communication and as
 the Servers are managing all the nodes in the region and performing scheduling.
 The memory and disk requirements are due to the fact that Nomad stores all state
-in memory and will store two snapshots of this data onto disk. Thus disk should
+in memory and will store two snapshots of this data onto disk, causing high IO write rate on high load clusters. Thus disk should
 be at least 2 times the memory available to the server when deploying a high
-load cluster.
+load cluster. When running on AWS prefer NVME or Provisioned IOPS SSD storage for data dir.
```
Comment (jippi) [inline website/source/guides/cluster/requirements.html.md]: we use 200 GB GP2 EBS volumes, and never see any kind of IO trouble, those are reasonable slow network drives. I think the problem is always defining what is `high` - we run clusters in size 100-200 nodes on 200 GB GP2 EBS volumes, even with a lot of periodic jobs, there are barely any CPU or disk usage worth mentioning

Comment (burdandrei) [reply website/source/guides/cluster/requirements.html.md]: 200 GB RBS on your masters? I saw clear IOPS write saturation on the leader with 64GB EBS.

File: website/source/guides/cluster/requirements.html.md [hunk 2]
```diff
@@ -12,14 +12,14 @@ description: |-
...
 
 **Nomad servers** may need to be run on large machine instances. We suggest
-having between 4-8+ cores, 16-32 GB+ of memory, 40-80 GB+ of disk and
+having between 4-8+ cores, 16-32 GB+ of memory, 40-80 GB+ of **fast** disk and
 significant network bandwidth. The core count and network recommendations are to
 ensure high throughput as Nomad heavily relies on network communication and as
 the Servers are managing all the nodes in the region and performing scheduling.
 The memory and disk requirements are due to the fact that Nomad stores all state
-in memory and will store two snapshots of this data onto disk. Thus disk should
+in memory and will store two snapshots of this data onto disk, causing high IO write rate on high load clusters. Thus disk should
```
Comment (preetapan) [inline website/source/guides/cluster/requirements.html.md]: wording suggestion - `onto disk, which causes high IO in busy clusters with lots of writes`.

File: website/source/guides/cluster/requirements.html.md (hunk 1 above)
Comment (jippi) [reply website/source/guides/cluster/requirements.html.md]: 200 GB GP2 EBS on masters yes, haven't seen any issues with that so far, we never see any dip on burst balance (we alert heavily on any burst balance usage)

Comment (burdandrei) [reply website/source/guides/cluster/requirements.html.md]: what is your raft.db size?

Comment (jippi) [reply website/source/guides/cluster/requirements.html.md]: around 150-250 MB

Comment (burdandrei) [reply website/source/guides/cluster/requirements.html.md]: Same here.
It looks like me, that nomad is not writing to it sequentially, but just dumps full in-memory state everytime, and this causes IO burst

Comment (jippi) [reply website/source/guides/cluster/requirements.html.md]: 64 GB GP2 EBS is 192 IOPS @ 128 MB/s
200 GB GP2 EBS is 600 IOPS @ 150 MB/s 

so quite a difference in throughput between our setups :)

**Pull request merged**